kie-gwthelper-maven-plugin/pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.drools</groupId>
+    <artifactId>droolsjbpm-integration</artifactId>
+    <version>7.19.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.kie</groupId>
+  <artifactId>kie-gwthelper-maven-plugin</artifactId>
+  <packaging>maven-plugin</packaging>
+
+  <name>KIE :: Gwt Helper Maven Plugin</name>
+
+  <prerequisites>
+    <maven>${maven.required.version}</maven>
+  </prerequisites>
+
+  <properties>
+    <maven.required.version>3.0</maven.required.version>
+    <plexus.component.annotations>1.7.1</plexus.component.annotations>
+    <plexus.utils.version>3.1.0</plexus.utils.version>
+    <maven.artifact.version>3.6.0</maven.artifact.version>
+    <maven.source.plugin.version>3.0.1</maven.source.plugin.version>
+    <maven.javadoc.plugin.version>3.0.1</maven.javadoc.plugin.version>
```

Comment:
we should use the same property name standards like elsewhere in kiegroup.

`version.<groupId>`

Reply:
btw for maven plugins we have already plugin management on top level in kie-parent or jboss-parent, so that would be good too to have them consistent with these versions.